services/src/main/java/io/druid/cli/CliRouter.java

```diff
@@ -65,7 +65,7 @@
  */
 @Command(
     name = "router",
-    description = "Experimental! Understands tiers and routes things to different brokers"
+    description = "Experimental! Understands tiers and routes things to different brokers, see http://druid.io/docs/latest/Router.html for a description"
```

Comment:
http://druid.io/docs/latest/development/router.html is more up to date.

Reply:
done